proposed/2021/DotnetNugetTrustedSigners.md

```diff
@@ -0,0 +1,239 @@
+# dotnet nuget trust command
+
+- Author Name [Kat Marchan (@zkat)](https://github.com/zkat)
+- Start Date 2021-03-04
+- GitHub Issue: https://github.com/NuGet/Home/issues/10634
+- GitHub PR: https://github.com/NuGet/Home/pull/10628
+
+## Summary
+
+This specifies the syntax and behavior for the `dotnet nuget trust` command.
+
+## Motivation
+
+This is part of a larger effort to reach full parity between `nuget` and
+`dotnet` for NuGet-related operations.
+
+## Explanation
+
+### Functional explanation
+
+#### `> dotnet nuget trust`
+
+```
+Provides the ability to manage the list of trusted signers.
```

Comment:
What do you think of adding a little more detail? If I'm a NuGet newbie, I probably won't know what "trusted signers" are for. For example:

```suggestion
Manage the trusted signers. By default, NuGet accepts all authors and repositories. You can restrict this to just authors or repositories you trust using this command.
```